File: Linguagem C#/Retangulo.cs/Retangulo.cs/retangulo.cs
// See https://aka.ms/new-console-template for more information
using System.Globalization;

internal class Program
{
    private static void Main(string[] args)
    {
        CultureInfo CI = CultureInfo.InvariantCulture;

        Console.Write("Base do retangulo: ");
        double Base = double.Parse(Console.ReadLine()!, CI);
        Console.Write("Altura do retangulo: ");
        double altura = double.Parse(Console.ReadLine()!, CI);
        double area = Base * altura;
        double perimetro = 2 * (Base + altura);
        double diagonal = Math.Sqrt(Base * Base + altura * altura);
        Console.WriteLine("AREA = " + area.ToString("F4", CI));
        Console.WriteLine("PERIMETRO = " + perimetro.ToString("F4", CI));
        Console.WriteLine("DIAGONAL = " + diagonal.ToString("F4", CI));
        
    }
}

File: Linguagem Python/EXs - Estrutura Sequencial PY/retangulo.py
import math

base = float(input("Base do retangulo: "))
altura = float(input("Altura do retangulo: "))
area = base*altura
print(f"Area = {area:.4f}")
print(f"Perimetro = {2*base + 2*altura:.4f}")
print(f"Diagonal = {math.sqrt(base**2 + altura**2):.4f}")
